featuretools/entityset/entity.py

```diff
@@ -84,7 +84,7 @@ def __repr__(self):
         repr_out = u"Entity: {}\n".format(self.id)
         repr_out += u"  Variables:"
         for v in self.variables:
-            repr_out += u"\n    {} (dtype: {})".format(v.id, v.dtype)
+            repr_out += u"\n    {} ".format(v.id)
```

Comment:
still print the dtype, just use `self._dtype_repr`

Reply:
Fixed.